core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2023,6 +2023,22 @@ package object config {
       .stringConf
       .createWithDefaultString("PWR")
 
+  private[spark] val EXECUTOR_DECOMMISSION_BATCH_INTERVAL =
+    ConfigBuilder("spark.executor.decommission.batchInterval")
+      .doc("Executors are decommissioned in batched to avoid overloading network bandwidth in" +
+        " migrating rdd and shuffle data. This config sets the interval between batches.")
+      .version("3.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefault(30000)
+
+  private[spark] val EXECUTOR_DECOMMISSION_BATCH_SIZE =
+    ConfigBuilder("spark.executor.decommission.batchSize")
+      .doc("Executors are decommissioned in batched to avoid overloading network bandwidth in" +
+        " migrating rdd and shuffle data. This config sets the interval between batches.")
```

Comment:
`the interval between batches` looks like a typo because this is a batch size, not an interval.

Reply:
Thanks for pointing it out. I have fix it.